app/decorators/provider_decorator.rb

```diff
@@ -1,13 +1,29 @@
 class ProviderDecorator < Draper::Decorator
   delegate_all
 
+  def short_address
+    address_lines.join(", ")
+  end
+
   def full_address
-    [object.address1, object.address2, object.address3, object.address4, object.postcode].select(&:present?).join("<br> ").html_safe
+    address_lines.map { |line| ERB::Util.html_escape(line) }.join("<br> ").html_safe
```

Comment:
Should this, and the other related fixes, be part of a separate PR?

Reply:
I can if we really want to. This is the only case of it on Find though and because I'm making other changes to this file as a result of refactoring I thought I might as well put it here. The commit for it is still marked with the appropriate card.

Reply:
Fine by me.